package.json

```diff
@@ -46,6 +46,8 @@
     "eslint-plugin-prettier": "4.2.1",
     "eslint-plugin-simple-import-sort": "10.0.0",
     "eslint-plugin-tsdoc": "0.2.17",
+    "fetch-mock": "9.11.0",
+    "fetch-mock-jest": "1.5.1",
```

Comment:
What's the difference between the 2 packages? Do we need both?

Reply:
https://www.npmjs.com/package/fetch-mock-jest

`fetch-mock-jest` wraps `fetch-mock`. I'm referencing `fetch-mock` to get the type in the `catalogs.test.ts` so the thought was if you're referencing it it should be in the package.json.
```typescript
import { FetchMockStatic } from 'fetch-mock'
import * as fetchMockJest from 'fetch-mock-jest'
```

Reply:
Gotcha, does `fetch-mock-jest` not export the necessary types?

Reply:
tbh, not sure. My thought is no, but I will look to see. Might not get to this until tomorrow.

Reply:
Sounds good, I'll approve the PR so it doesn't block you from merging

Reply:
I found a hack around it by doing `typeof fetchMockJest` which returns more info, which technically might be more correct anyway. I removed the import.

Reply:
Nice!